src/components/heatmap/heatmapAnalysisForm/HeatmapAnalysisForm.jsx

```diff
@@ -0,0 +1,147 @@
+// @flow
+import * as React from "react";
+import Form from "../../form/Form";
+import Box from "grommet/components/Box";
+import FormField from "grommet/components/FormField";
+import FormFields from "grommet/components/FormFields";
+import NumberInput from "grommet/components/NumberInput";
+import Value from "grommet/components/Value";
+import { inputHandler } from "../../../utils/InputHandler";
+import type { AllInputTypes } from "../../../utils/InputHandler";
+import HeatmapData from "../../../models/HeatmapData";
+import type { HeatmapMode } from "../../../types/Heatmap";
+
+type Props = {
+	heatmapData: HeatmapData,
+	maxLuxValue: number,
+	heatmapMode: HeatmapMode,
+	onChange: (string, AllInputTypes) => void
```

Comment:
wrong signature: `onChange: (key: string, value: AllInputTypes) => void`

Reply:
the parameter names are optional: https://flow.org/en/docs/types/functions/#toc-function-types